src/box/memtx_tx.h

```diff
@@ -151,6 +150,13 @@ struct memtx_story {
 	 * Number of indexes in this space - and the count of link[].
 	 */
 	uint32_t index_count;
+	/** Status of story for memory manager. */
+	int pin_status;
```

Comment:
Please don't use `int` for enum values. There are exceptions though, but they are seldom and this is not the one.

Reply:
The meaning is not clear.

Reply:
Replaced with enum

Reply:
Extended the comment